tests/test_xarray.py

```diff
@@ -1298,6 +1298,14 @@ def func(a, b):
     assert_array_equal(func(data, data2), [[0, 1, 2], [0, 0, 0], [0, 0, 0]] * units('N m'))
 
 
+def test_preprocess_and_wrap_broadcasting_error():
+    """Test that decorator with bad arguments specified to broadcast errors out."""
+    with pytest.raises(ValueError):
+        @preprocess_and_wrap(broadcast=('a', 'c'))
+        def func(a, b):
+            return a + b
```

Comment:
Ooh, don't want an un-run line in tests. How about:
```suggestion
            pass
```